### PR TITLE
docs: automatic yanking using nvim-osc52 and enabling plugin only in SSH

### PR DESCRIPTION
**Pull request by Gelio**
Hey! Thanks for a great plugin. I just spent a few hours trying to debug [vim-oscyank](https://github.com/ojroques/vim-oscyank) using tmux 3.3a and kitty and nvim-osc52 made it finally work :tada: 

I want to contribute some quality-of-life improvements to the suggested configuration. I have been using them for some time and they worked well for me. I hope they are useful to other users of the plugin as well

### CODE REVIEW

**Comment by ojroques** [comment]
Hello!

First thank you for your PR :) So for the first case (Automatically yanking using nvim-osc52) it's simpler to do it that way: https://github.com/ojroques/nvim-osc52/issues/1#issuecomment-1193984122.
TL;DR: 
```lua
function copy()
  if vim.v.event.operator == 'y' and vim.v.event.regname == '+' then
    require('osc52').copy_register('+')
  end
end

vim.api.nvim_create_autocmd('TextYankPost', {callback = copy})
```
I've ommitted `require('osc52').copy_register` in the README, I'll add it later.

**Comment by Gelio** [comment]
I didn't know about `vim.v.event`. Thanks! It indeed makes it simpler!

I can mention it in this PR, if you believe it is relevant. Let me know what changes you want me to make to this PR, if you plan on merging it rather than documenting it yourself

**Comment by ojroques** [comment]
I've updated the README with an advanced usage section that includes your use case. Thanks!

**Comment by Gelio** [comment]
Thanks!

There's still the section about only enabling the plugin when in an SSH session. I don't see it mentioned in the readme. I thought that would also be useful, but I'll leave it up to you.
